web/app/src/middleware/home.ts

```diff
@@ -26,7 +26,8 @@ export async function middleware(
     //   return NextResponse.redirect(loginUrl);
     // }
     if (url.pathname === '/') {
-      return NextResponse.redirect(new URL('/welcome', request.url));
+      // return NextResponse.redirect(new URL('/welcome', request.url));
```

Comment:
建议添加注释说明为什么使用 rewrite 而不是 redirect，以便其他开发者理解这个设计决策

```suggestion
      // 使用 rewrite 而不是 redirect 以保持 URL 不变，提供更好的用户体验
      return NextResponse.rewrite(new URL('/welcome', request.url));